eraser-map.yaml

```diff
@@ -2204,8 +2208,9 @@ layers:
         data: { source: osm, layer: buildings }
         filter: 
             all:
-                - { $zoom: { min: 20 }, kind: address }
-                - { $zoom: { min: 20 }, [ kind: true, addr_housenumber: true, name: false ] }
+                - $zoom: { min: 20 }
```

Comment:
I'm not clear of the intent here and whether the old or new syntax achieves it :)  @nvkelso @blair1618 

My read is that this should be:

```
filter:
   $zoom: { min: 20 }
   any:
      - kind: address
      - { kind: true, addr_housenumber: true, name: false }
```
Zoom is >= 20, AND EITHER kind = address OR (kind and addr_housenumber have values, but name does not have a value).